v-dev.sql

```diff
@@ -0,0 +1,55 @@
+--Drop the column practice_type_id if it exists
+ALTER TABLE openchpl.test_functionality DROP COLUMN IF EXISTS practice_type_id;
+--Add the column practice_type_id to terst_functionality table
+ALTER TABLE openchpl.test_functionality ADD COLUMN practice_type_id bigint DEFAULT null;
+--Add the FK relation between test_functionality and practice_type
+ALTER TABLE openchpl.test_functionality 
+		ADD CONSTRAINT practice_type_fk 
+		FOREIGN KEY (practice_type_id)
+        REFERENCES openchpl.practice_type (practice_type_id) MATCH SIMPLE
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION;
+
+--Drop the column certification_criterion_id if it exists
+ALTER TABLE openchpl.test_functionality DROP COLUMN IF EXISTS certification_criterion_id;
+--Add the column certification_criterion_id to terst_functionality table
+ALTER TABLE openchpl.test_functionality ADD COLUMN certification_criterion_id bigint DEFAULT null;
+--Add the FK relation between test_functionality and certification_criterion_id
+ALTER TABLE openchpl.test_functionality 
+		ADD CONSTRAINT certification_criterion_fk 
+		FOREIGN KEY (certification_criterion_id)
+        REFERENCES openchpl.certification_criterion (certification_criterion_id) MATCH SIMPLE
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION;
+		
+--Update the appropriate test_functionality records with practice type and certification criterion		
+UPDATE openchpl.test_functionality SET practice_type_id = 1, certification_criterion_id = 65 WHERE test_functionality_id = 6; 
```

Comment:
Same comment as above to get the IDs from another select. I just think it's a little more human readable and eliminates the possibility that a different instance of our database might have a different ID for any of these criteria (they don't but still).